dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -97,6 +98,7 @@ public class ConnectorArguments extends DefaultArguments {
     public static final String OPT_DATABASE = "database";
     public static final String OPT_SCHEMA = "schema";
     public static final String OPT_ASSESSMENT = "assessment";
+    public static final String OPT_MAX_TABLESIZEV_ROWS = "max-tablesizev-rows";
```

Comment:
Other dialect-specific flags incorporate the dialect in their name, e.g. `OPT_ORACLE_SID` -> `--oracle-sid`.
Can we have `OPT_TERADATA` as prefix?